src/app/app.component.ts

```diff
@@ -30,6 +31,10 @@ export class App {
         this.configureStatusBar();
         this.configureAccessibility();
         this.loadAppInfo();
+
+        if (cordova && cordova.plugins && cordova.plugins.DisableShakeToEdit) {
+          cordova.plugins.DisableShakeToEdit.pluginInitialize();
```

Comment:
@amobarak Pretty sure we don't need this, the plugin gets initialised automatically

Reply:
@Cavalier07 Any ideas?

Reply:
try it without?